Electron/Core/RoveProtocol/Rovecomm.ts

```diff
@@ -199,24 +239,16 @@ class Rovecomm extends EventEmitter {
         port = DATAID[i].Port
         dataType = DATAID[i].Commands[dataIdStr].dataType
         dataId = DATAID[i].Commands[dataIdStr].dataId
-        found = true
         break
       }
     }
-    if (found === false) {
-      this.emit(
-        "all",
-        `Attempting to send packet with DataId: ${dataIdStr} and data ${data} but dataId was not found. Packet not sent.`
-      )
-      return
-    }
```

Comment:
Removal of the found Boolean and its handling removes a safety for if a packet is attempted to be sent with an invalid dataId